_layouts/ontology_detail.html

```diff
@@ -278,6 +278,20 @@ <h1>
           >{{page.exampleClass}}</a
         >
       </dd>
+      {% endif %} {% if page.issue_requested %}
+      <dt>New Ontology Request</dt>
+      <dd>
+        <a href="https://github.com/OBOFoundry/OBOFoundry.github.io/issues/{{ page.issue_requested }}">
+          #{{ page.issue_requested }}
+        </a>
+      </dd>
+      {% endif %} {% if page.pull_request_added %}
+      <dt>PR Added</dt>
+      <dd>
+        <a href="https://github.com/OBOFoundry/OBOFoundry.github.io/pull/{{ page.pull_request_added }}">
+          #{{ page.pull_request_added }}
+        </a>
+      </dd>
```

Comment:
Just to merge this a bit faster, can we atm not display the `<dt>PR Added</dt>` section? Its in a very prominent position, and I am assuming that people will push a bit against it..

Reply:
Actually forget it, I will do it.